addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/ExtensionWebSocket.java

```diff
@@ -491,12 +545,27 @@ public String getDescription() {
 	 * 
 	 * @return the script icon, never {@code null}.
 	 */
-	private static ImageIcon getScriptIcon() {
-		if (scriptIcon == null) {
-			scriptIcon = new ImageIcon(
+	private static ImageIcon getScriptSenderIcon() {
+		if (scriptSenderIcon == null) {
+			scriptSenderIcon = new ImageIcon(
 					ExtensionWebSocket.class.getResource("/org/zaproxy/zap/extension/websocket/resources/script-plug.png"));
 		}
-		return scriptIcon;
+		return scriptSenderIcon;
+	}
+	
+	/**
+	 * Gets the icon for scripts types.
+	 * <p>
+	 * Should be called/used only when in view mode.
+	 *
+	 * @return the script icon, never {@code null}.
+	 */
+	private ImageIcon getScriptPassiveScanIcon() {
+		if (scriptPassiveScanIcon == null) {
+			scriptPassiveScanIcon = new ImageIcon(Exception.class.getResource(
+					"/resource/icon/fugue/plug-connect.png")); //TODO: Find another Image
```

Comment:
Script+Icons is uses by WebSocket Sender. What about that one? ![plug--passive](https://user-images.githubusercontent.com/10010230/57689738-c2d56a80-7648-11e9-9891-821dae0d8f3f.png)

Reply:
I assume that's eyeball & plug? I can't make out the plug part it just seems like a black blob. If it can be made more clear then great, at the same time I realize this isn't a gfx design project so go ahead with it, we can always replace it later I guess...